### PR TITLE
Add linting script and lint files

### PR DESCRIPTION
**Pull request by martinratinaud**
Fixes https://github.com/ambanum/OpenTermsArchive/issues/833

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -280,6 +280,16 @@ Since this operation fetches documents and could be long, you can also validate
 npm run test:schema [$service_id [, $service_id …]]
 ```
 
+#### Linting
+
+In order to ensure consistency across declarationsan repo, all declarations files have to be linted (formatted) properly and homogeneously.
```
Comment (MattiSG) [inline CONTRIBUTING.md:285]:
```suggestion
In order to ensure consistency across declarations, all declarations files have to be formatted homogeneously.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -280,6 +280,16 @@ Since this operation fetches documents and could be long, you can also validate
...
+
+In order to achieve this, you can use the following script
```
Comment (MattiSG) [inline CONTRIBUTING.md:287]:
```suggestion
In order to achieve this, you can use the following script:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -280,6 +280,16 @@ Since this operation fetches documents and could be long, you can also validate
...
+
+```
+npm run lint [, $service_id …]
```
Comment (MattiSG) [inline CONTRIBUTING.md:290]:
```suggestion
npm run lint [ $service_id …]
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
CI should fail upon linting failure.

**Comment by martinratinaud** [comment]
> CI should fail upon linting failure.

It will when `json_lint` branch is merged on master on OTA core

Tests wlll pass on this branch when all declarations in contrib-declarations are fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (martinratinaud) [reply CONTRIBUTING.md]: I reused what was above in the document (written by @Ndpnt).

I believe the goal of this is to explain that you can use this command with OR without any service_id.
Your suggestion does not reflect this possibility I believe. So I will leave it like this unless you believe it's really not understandable.

Let me know

Comment (MattiSG) [reply CONTRIBUTING.md:290]: The “optional” part of a command line argument is by convention made clear with the brackets 🙂

The most conventional way to show that there can be 0-to-many arguments would be:

```suggestion
npm run lint [$service_id [, $another_service_id … ]]
```

Comment (MattiSG) [reply CONTRIBUTING.md:290]: See https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text

Comment (MattiSG) [reply CONTRIBUTING.md:290]: If you want, I'm happy for us to move towards explicitly using http://docopt.org as a reference, but let's make this into a dedicated PR that also updates the CONTRIBUTING 🙂

Comment (martinratinaud) [reply CONTRIBUTING.md:290]: thanks. I've harmonized this then

Comment (MattiSG) [reply CONTRIBUTING.md:290]: I've created https://github.com/ambanum/OpenTermsArchive/issues/848 accordingly, thanks for raising this point! 😊

**Comment by martinratinaud** [comment]
MErged after talking with @MattiSG

**Pull request merged**